src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -311,6 +338,20 @@ const AuthoritiesSearch = ({ children }) => {
             </Button>
           </form>
 
+          <Accordion
```

Comment:
Please move this section to `<SearchFilters>` component if possible, I'd like to have all filters together in one component